### DISCUSSION TITLE
I'm just curious what happened to development. Is this project still being worked on?

### DISCUSSION THREAD

**Discussion by NYANLAUNCHER** [wren-lang/wren#1184] [General]
Here's the body to this discussion, of which I didn't need.

**Comment by minirop** [comment]
here: https://github.com/wren-lang/wren/issues/1180

**Reply by NYANLAUNCHER** [reply]
thanks

**Reply by YatesRocks** [reply]
Also here: https://github.com/wren-lang/wren/discussions/1167#discussion-5131816
